src/server/pfs/server/compaction.go

```diff
@@ -283,10 +336,8 @@ func processCompactTask(ctx context.Context, storage *fileset.Storage, task *Com
 			return err
 		}
 		pathRange := &index.PathRange{
-			Lower:      task.PathRange.Lower,
-			LowerDatum: task.PathRange.LowerDatum,
```

Comment:
Can you remind me why we're removing the Datum Range? because of `[Lower, Upper)` ?

Reply:
Yeah, it is no longer necessary.